runtime/runtime/src/prefetch.rs

```diff
@@ -157,12 +164,12 @@ impl TriePrefetcher {
         self.prefetch_api.clear_data();
     }
 
-    fn prefetch_trie_key(&self, trie_key: TrieKey) -> Result<(), ()> {
+    fn prefetch_trie_key(&self, trie_key: TrieKey) -> Result<(), PrefetchQueueFullError> {
         let queue_full = self.prefetch_api.prefetch_trie_key(self.trie_root, trie_key).is_err();
```

Comment:
If we make the change to error type above, then `self.prefetch_api.prefetch_trie_key()` should be returning the type.

Reply:
good point, done!